staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -106,9 +108,10 @@ func handleRootHealthz(checks ...HealthzChecker) http.HandlerFunc {
 		failed := false
 		var verboseOut bytes.Buffer
 		for _, check := range checks {
-			if check.Check(r) != nil {
+			if err := check.Check(r); err != nil {
 				// don't include the error since this endpoint is public.  If someone wants more detail
 				// they should have explicit permission to the detailed checks.
+				utilruntime.HandleError(fmt.Errorf("healthz check %v failed: %v", check.Name(), err))
```

Comment:
IMO this is not an error in the normal sense, i.e. it shouldn't be passed to `HandleError`. A "not yet ready" will also end up in this branch, won't it?

Reply:
Ok, so a normal `glog.Errorf` / `glog.Infof` ?
@sttts @tallclair

Reply:
A glog.Infof with a high verbosity level.